### ISSUE TITLE
Adding DISPLAY directions for COVSUMMARY

### ISSUE THREAD

**Issue by cusackjm** [episphere/questionnaire#100]
I would like to edit the questionnaire document and add display directions for COVSUMMARY. "*DISPLAY IF COV3, COV4, COV6, COV8A, COV8B, COV8C, COV12 WERE DISPLAYED TO THE RESPONDANT]"

Similar to PREGSUMMARY: 
![image](https://user-images.githubusercontent.com/85582895/167901756-563881a8-7c87-4534-8dbb-f97adff425ff.png)

I'm assuming this is possible because it was in PREGSUMMARY, but will wait to add the DISPLAY text to the questionnaire document until I get confirmation that this is possible.

Also wanted to note the open issue for the SUMMARY spacing https://github.com/episphere/questionnaire/issues/86

**Comment by cusackjm** [comment]
Display directions have been added to the questionnaire documents and are denoted in tracked changes

**Comment by joshid-ims** [comment]
@cusackjm So the individual details in the summary should be displayed of the corresponding question was displayed, right? And does that mean the we need to check if the question was only displayed and not worry about how they answered it (or it is NR)? So basically the summary detail for that question will be displayed but left blank if it is NR? 

Also, are we displaying only A, B, C responses from question 8 or all of them?

**Comment by joshid-ims** [comment]
@cusackjm, sorry one question. I thought I would be able to show blank response for NR but actually data (or empty string) is not stored in the background if the response is NR (in case of text boxes, it can be compared with ""). 

So for NR, is it ok to not display the summary option?

**Comment by cusackjm** [comment]
@joshid-ims Yes, if a question does not have a response to it, it is ok to not show it in the summary

**Comment by joshid-ims** [comment]
Thanks

**Comment by cusackjm** [comment]
fixed in 5/20 testing but leaving open because of spacing

**Comment by KELSEYDOWLING7** [comment]
Seems fixed in 6/8 testing

**Comment by cunnaneaq** [comment]
Agreed - fixed, but still has spacing issue
![Screen Shot 2022-06-08 at 2 17 15 PM](https://user-images.githubusercontent.com/83971268/172688338-abc5d38c-5977-43d2-833e-e819abe7c23b.png)

**Comment by cusackjm** [comment]
> @cusackjm, sorry one question. I thought I would be able to show blank response for NR but actually data (or empty string) is not stored in the background if the response is NR (in case of text boxes, it can be compared with "").
> 
> So for NR, is it ok to not display the summary option?

Is it possible to still show the question and fill "no response" for questions  the participant skipped? (so that ppl could possibly go back and enter an answer?)
